sites/main-page-title/includes/main-page-title.html

```diff
@@ -0,0 +1,12 @@
+{% if include %}
+	{% if include.dir == 'ltr' or include.dir == 'rtl' %}
+		<h1 property="name"
+		{%- if include.rdfa -%}
+		typeof="WebPageElement"
+		{%- endif -%}
+		{%- if include.id -%}
+		id="wb-cont"
+		{%- endif -%}
+		dir="{{ include.dir }}">{{ page.title }}</h1>
```

Comment:
We don't need to include the direction `dir`. That is usually/should be done on an higher level like the `<html>` element.

We should allow any general attribute and we should document what is essential or what is optional like it apply some customization.